config.yml.tpl

```diff
@@ -8,7 +8,7 @@ analyzers:
 
 providers:
   github:
-    comment_footer: '_If you have feedback about this comment, please, [tell us](%s)._'
+    comment_footer: "_If you have feedback about this comment made by the analyzer {{.Name}}{{with .Feedback}}, please, [tell us]({{.}}){{end}}._"
```

Comment:
What do you get if feedback is not defined? 

> If you have feedback about this comment made by the analyzer style-analyzer.

The sentence feels unfinished.

Reply:
If the repo maintainers want to override the feedback URL, why not letting them do it?
Maybe they misconfigured the analyzer with their local `.lookout.yml`, and they want to be informed about that. Or maybe that way the repo contributors can ask the repo maintainer to disable the analyzer in that repo because it produces no valuable comments...

Reply:
goot catch; addressed by https://github.com/src-d/lookout/pull/495/commits/d34cab394be1372f18a3706b99005e300ae7380e